stdlib/list.gr

```diff
@@ -329,3 +329,57 @@ let join = (separator: String, items: List<String>) => {
     Some(s) => s,
   }
 }
+
+/**
+ * Reverses the first list and appends the second list to the end.
+ * 
+ * @param list1: The list to reverse
+ * @param list2: The list to append
+ * @since v0.4.5
+ */
+let rec revAppend = (list1, list2) => { 
```

Comment:
@phated Did you have any thoughts on `revAppend` vs `reverseAppend`? I personally like `revAppend` but `reverseAppend` might be more clear. (But I really like `revAppend`.)

Reply:
No opinion really. You know I like verbosity for clarity, but I don't think `rev` gets confused with anything else (can you think of anything?). If you are **really** like `revAppend` then let's stick with it for now.

Reply:
Works for me!